pkg/acsengine/const.go

```diff
@@ -90,6 +102,12 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"dnsmasq":      "k8s-dns-dnsmasq-amd64:1.13.0",
 		"pause":        "pause-amd64:3.0",
 		"windowszip":   "v1.6.0intwinnat.zip",
+		"nodestatusfreq":   "10s",
+		"nodegraceperiod":   "40s",
+		"podeviction":   "5m0s",
+		"routeperiod":   "10s",
```

Comment:
Sounds good. I'd just want to make sure when we add parameters /use them in the custom data that nothing breaks. I don't imagine it would, just would want to verify.